addons/modules/XEH_postInit.sqf

```diff
@@ -45,7 +45,7 @@ if (isServer) then {
 }] call CBA_fnc_addEventHandler;
 
 [QGVAR(applyWeather), {
-    params ["_forced", "_overcast", "_rain", "_lightning", "_rainbow", "_waves", "_wind", "_gusts", "_fog"];
+    params ["_forced", "_overcast", "_rain", "_lightning", "_rainbow", "_waves", "_wind", "_gusts", "_fog", ["_precipitation", -1]];
```

Comment:
Don't need default value and move after `_rain` (in order that it shows up in module).

Reply:
> Don't need default value and move after `_rain` (in order that it shows up in module).

Isn't backwards compatibility a concern?

Reply:
Should `_precipitation` be changed to `_precipitationType` and `_rain` to `_precipitation` ?

Reply:
I'd say just `_precipitationType`.